trollmoves/dispatcher.py

```diff
@@ -318,7 +321,19 @@ def run(self):
                     continue
                 destinations = self.get_destinations(msg)
                 if destinations:
-                    success = dispatch(msg.data['uri'], destinations)
+                    # Check if the url are on another host:
+                    url = urlparse(msg.data['uri'])
+                    if url.scheme != '':
+                        netloc = url.netloc
+                        if netloc != self.host:
```

Comment:
This doesn't always work. See here how it is solved:
https://github.com/pytroll/trollmoves/blob/master/trollmoves/server.py#L351-L352

Maybe a function could be made for this and used in both places ?